src/styles/index.scss

```diff
@@ -1,3 +1,149 @@
+$main-accent: #060b35;
+$secondary: #616070;
+$blue_accent: #00acdc;
+
+html {
+  background-color: darkgray;
+  font-family: "Roboto", sans-serif;
+}
+
 body {
   margin: 0;
+  box-sizing: border-box;
+}
+
+.card {
+  box-sizing: border-box;
+  display: flex;
+  flex-direction: column;
+  margin: 50px auto;
+  width: 200px;
+  height: 406px;
+  border-radius: 5px;
+  border: 1px solid #f3f3f3;
+  background-color: #fff;
+
+  &__title {
+    margin: 0 auto;
+    font-size: 12px;
+    font-weight: 500;
+    line-height: 18px;
+    width: 166px;
+    color: $main-accent;
+    margin-bottom: 4px;
+  }
+
+  &__image {
+    margin: 0 auto;
+    margin-top: 32px;
+    margin-bottom: 40px;
+    width: 160px;
+    height: 134px;
+  }
+
+  &__code {
+    margin-left: 16px;
+    font-size: 10px;
+    font-weight: 400;
+    line-height: 14px;
+    color: $secondary;
+    margin-bottom: 16px;
+  }
+}
+
+.block {
+  box-sizing: border-box;
+  display: flex;
+  width: 166px;
+  justify-content: space-between;
+  align-items: flex-end;
+  margin: 0 auto;
+  padding: 0;
+  margin-bottom: 24px;
+
+  &__stars {
+    display: flex;
+    justify-content: center;
+
+    .star {
+      width: 16px;
+      height: 16px;
+      background-repeat: no-repeat;
+      background-image: url(../images/star.svg);
+      background-position: center;
+    }
+    .star:not(:last-child) {
```

Comment:
devide styles by files, stars to stars.scss, card to card.scss, etc, common styles keep in index.scss